packages/contracts-bedrock/contracts/test/L2StandardBridge.t.sol

```diff
@@ -57,12 +127,91 @@ contract L2StandardBridge_Test is Bridge_Initializer {
         // Alice has 100 L2Token
         deal(address(L2Token), alice, 100, true);
         assertEq(L2Token.balanceOf(alice), 100);
+        uint256 nonce = L2Messenger.messageNonce();
+        bytes memory message = abi.encodeWithSelector(
+            StandardBridge.finalizeBridgeERC20.selector,
+            address(L1Token),
+            address(L2Token),
+            alice,
+            alice,
+            100,
+            hex""
+        );
+        uint64 baseGas = L2Messenger.baseGas(message, 1000);
+        bytes memory withdrawalData = abi.encodeWithSelector(
+            CrossDomainMessenger.relayMessage.selector,
+            nonce,
+            address(L2Bridge),
+            address(L1Bridge),
+            0,
+            1000,
+            message
+        );
+        bytes32 withdrawalHash = Hashing.hashWithdrawal(
+            Types.WithdrawalTransaction({
+                nonce: nonce,
+                sender: address(L2Messenger),
+                target: address(L1Messenger),
+                value: 0,
+                gasLimit: baseGas,
+                data: withdrawalData
+            })
+        );
+
+        vm.expectEmit(true, true, true, true);
+        emit ERC20BridgeInitiated(address(L2Token), address(L1Token), alice, alice, 100, hex"");
+
+        vm.expectEmit(true, true, true, true);
+        emit MessagePassed(
```

Comment:
For this event, do you mind using the kwargs like syntax? ie `({ key: value, ...})`

Reply:
Very good nit!

Reply:
Very good nit!